spec/controllers/admin/videos_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'spec_helper'
+
+describe Admin::VideosController do
+  it {is_expected.to use_before_action(:require_user)}
```

Comment:
I wouldn't do this here. I'm not sure it actually works like this. Even if it did what you really want to do is test the results (that the user was redirect) not the implementation (that `require_user`) was used.